delta-api/src/main/java/io/cdap/delta/api/assessment/TableNotFoundException.java

```diff
@@ -25,7 +25,19 @@ public TableNotFoundException(String database, String table, String errorMessage
     super(String.format("Table '%s' in database '%s' was not found. %s", table, database, errorMessage));
   }
 
+  public TableNotFoundException(String database, String schema, String table, String errorMessage) {
+    super(String.format("Table '%s' in database '%s' and schema '%s' was not found. %s", table, database, schema,
+      errorMessage));
+  }
+
   public TableNotFoundException(String database, String table, String errorMessage, Throwable cause) {
     super(String.format("Table '%s' in database '%s' was not found. %s", table, database, errorMessage), cause);
```

Comment:
Also better call the variant `this(database, schema, table, errorMessage, cause)` from here.